addons/account/models/account_move.py

```diff
@@ -2304,6 +2304,25 @@ def write(self, vals):
 
         return res
 
+    def check_move_sequence_chain(self):
+        return self.filtered(lambda move: move.name != '/')._is_end_of_seq_chain()
+
+    @api.ondelete(at_uninstall=False)
+    def _unlink_forbid_parts_of_chain(self):
+        """ For a user with Billing/Bookkeeper rights, when the fidu mode is deactivated,
+        moves with a sequence number can only be deleted if they are the last element of a chain of sequence.
+        If they are not, deleting them would create a gap. If the user really wants to do this, he still can
+        explicitly empty the 'name' field of the move; but we discourage that practice.
+        If a user is a Billing Administrator/Accountant or if fidu mode is activated, we show a warning,
+        but they can delete the moves even if it creates a sequence gap.
+        """
+        if not (self.user_has_groups('account.group_account_manager') or self.company_id.quick_edit_mode)\
+           and not (self._context.get('force_delete') or self.check_move_sequence_chain()):
```

Comment:
yes, the intent is only to trigger if all conditions are false: 
 - the user does not have the appropriate rights
 - the fidu mode is off
 - there is no `force_delete` in the context
 - the move is not the last in the sequence chain. 
 
If any of the above is true, we should allow deleting the move.
 It was easier for me conceptually to split them into two related groups, because the first part checks the rights, and the second checks the rest. But I can change that, sure

Reply:
ping @xmo-odoo